src/bsp/tls/crypto-algorithms/README

```diff
@@ -0,0 +1,3 @@
+Files are from Brad Conte at: https://github.com/B-Con/crypto-algorithms
+Files are public domain
+Files have been slightly modified to adjust necessary #include files for compilation
```

Comment:
:+1: but according to the page https://github.com/B-Con/crypto-algorithms I think is ok leaving this.

Especially because according to the conversation with the Platform Team (GTyukas, JKlemm, LKorossy) the actual implementation of FINGERPRINT will be per customer / use case. It may be any checksum (MD5, sha256, CRC code, ...). Which means it won't be part of the Xively C Client but the Application. Probably this will be another BSP module: xi_bsp_update_fingerprint.h or just xi_bsp_update.h (I would like to rename the xi_bsp_sft.h to xi_bsp_update.h if you also agree).